languages/es_es/Settings/Vtiger.php

```diff
@@ -353,6 +353,19 @@
 	'LBL_COMPANY_LOGO_IN_SYSTEM' => 'Logotipo de la empresa visible en el sistema',
 	'LBL_COMPANY_LOGO_IN_LOGIN' => 'Logotipo de la empresa visible en el panel de inicio de sesión',
 	'LBL_HEIGHT_LOGO' => 'Altura del logotipo',
+	//Pluralized
+	'PLU_USERS_0' => 'Usuario',
+	'PLU_USERS_1' => 'Usuarios',
+	'PLU_USERS_2' => 'Usuarios',
```

Comment:
Ok thanks for the information, I got it from the pl_pl language. If it is not used, I'll remove them from there too.
#4609